Source/WebKit/UIProcess/RemoteLayerTree/RemoteAcceleratedEffectStack.h

```diff
@@ -59,9 +61,14 @@ class RemoteAcceleratedEffectStack final : public WebCore::AcceleratedEffectStac
 
     WebCore::AcceleratedEffectValues computeValues(MonotonicTime now) const;
 
+#if PLATFORM(MAC)
+    const WebCore::FilterOperations* longestFilter() const;
```

Comment:
Unclear what "longest" means

Reply:
Agreed, I'll rename to `longestFilterList()`.